lib/tasks/migrate_fedora.rake

```diff
@@ -163,4 +163,18 @@ namespace :louisville do
       end
     end
   end
+
+  desc 'Get Bulkrax Importer File names'
+  task get_importer_file_names: [:environment] do
+    # ************************************************************************
+    # This task simply returns the names of all the csv files that have been imported.
+    # ************************************************************************
+    AccountElevator.switch!(Account.first.cname)
+    importer_ids = Bulkrax::Importer.pluck(:id)
+    importer_ids.each do |importer_id|
+      importer = Bulkrax::Importer.find(importer_id)
+      fpath = importer[:parser_fields]['import_file_path'].split('/').last
+      puts "#{importer.name}: #{fpath}"
```

Comment:
1. Can we include the importer's ID too? 
2. Should we put this in a `tmp/` file? The only reason to do so would be if the printed lines get broken up by logs, e.g. 

```log
D -- [ksjdha;kskjdf;asdk] DEBUG : SELECT * FROM importers.id WHERE ...
Test Importer: tmp/uploads/my_csv.csv
D -- [ksjdha;kskjdf;asdk] DEBUG : SELECT * FROM importers.id WHERE ...
Second Importer: tmp/uploads/second.csv
```

Reply:
latest commit should address both of those suggestions! :)